yolapy/models/user.py

```diff
@@ -0,0 +1,54 @@
+from yoconfig import get_config
+from yolapy.services import Yola
+
+
+class User(object):
+
+    """Yola User."""
+
+    def __init__(self, active=False, deleted=None, email=None, id=None,
+                 name=None, partner_id=None, preferences=None,
+                 signup_date=None, surname=None):
+        """Sets all user attributes and inits a Yola client."""
+        self._partner_id = partner_id
+        self.active = active
+        self.deleted = deleted
+        self.email = email
+        self.id = id
+        self.name = name
+        self.preferences = preferences or {}
+        self.signup_date = signup_date
+        self.surname = surname
+
+        self.client = Yola()
+
+    @property
+    def partner_id(self):
+        """Use the auth user of the Yola client as the default partner ID."""
+        return self._partner_id or get_config('yola')['auth'][0]
```

Comment:
From an uninformed POV it is not immediately apparent what `['auth'][0]` is, so throwing it in method allowed me to a nice to add documentation.

An extra bonus (albeit negligible), moving logic out of the init.  My favorite initializers are ones that are free of logic and side-effects (unless the class name specifically denotes such things).

Side note: looking at my init I'm not entirely sure `self.client = Yola()` should be in there.  Maybe @blaix tell me it's wrong and show me a better way to handle the dependancy.